src/main/kotlin/car/business/Cars.kt

```diff
@@ -0,0 +1,33 @@
+package car.business
+
+import kotlin.random.Random
+import kotlin.streams.toList
+
+class Cars(amount: Int, random: Random = Random) {
+    private val RANDOM_MAX_SIZE = 10
+
+    private val cars: List<Car>
+    private val random: Random
+
+    init {
+        cars = ArrayList(amount)
+        for (i in 0 until amount) {
+            cars.add(Car())
+        }
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
        cars = (1..amount).map { Car() }
```

Reply:
와........... 감탄이 나왔습니다.....